### PR TITLE
Cut v0.22.3 release

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

**Comment by r2d4** [comment]
Are we going to release an iso also?  We'll need a PR to bump that version first

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Minikube Release Notes
 
+# Version 0.22.3 - 10/3/2017
+* Update dnsmasq to 1.14.5 [2022](https://github.com/kubernetes/minikube/pull/2022)
```
Comment (r2d4) [inline CHANGELOG.md]: I think this release will also includes the windows fixes

Comment (r2d4) [reply CHANGELOG.md]: Can you add the new PRs we just merged?  

* Components (apiserver, controller-manager, scheduler, kubelet) can now be configured in the kubeadm bootstrapper can now be configured with the --extra-config flag
* Kubeadm bootstrapper updated to work with Kubernetes v1.8.0
* OpenAPI registration fix cherry-picked for compatibility with kubectl v1.8.0

Comment (aaron-prindle) [reply CHANGELOG.md]: done, tests have finished

**Pull request merged**